hassio/src/hassio-main.ts

```diff
@@ -131,13 +136,15 @@ class HassioMain extends ProvideHassLitMixin(HassRouterPage) {
       return;
     }
 
-    const [supervisorInfo, hostInfo, hassInfo] = await Promise.all([
+    const [supervisorInfo, hostInfo, hassOsInfo, hassInfo] = await Promise.all([
       fetchHassioSupervisorInfo(this.hass),
       fetchHassioHostInfo(this.hass),
+      fetchHassioHassOsInfo(this.hass),
       fetchHassioHomeAssistantInfo(this.hass),
     ]);
     this._supervisorInfo = supervisorInfo;
     this._hostInfo = hostInfo;
+    this._hassOsInfo = hassOsInfo;
```

Comment:
Since it's only needed in the updater card, maybe we should just fetch that data there ?